Cargo.toml

```diff
@@ -8,24 +8,24 @@ authors = ["adorsys GmbH Co. KG"]
 
 [workspace]
 members = [
-    "crates/did-utils",
-    "crates/keystore",
+    "crates/web-plugins/didcomm-messaging/src/keystore",
+    "crates/web-plugins/didcomm-messaging/src/did-utils",
     "crates/plugin-api",
-    "crates/plugins/did-endpoint",
-    "crates/plugins/mediator-coordination",
-    "crates/plugins/oob-messages",
-    "crates/database",
+    "crates/web-plugins/did-endpoint",
```

Comment:
Place the crates directly at the root of the **didcomm-messaging** folder rather than the sub **src** folder. The **didcomm-messaging** folder will not be a crate itself, but rather an abstract folder containing all the DIDComm related stuffs.

Reply:
Okay, I thought of that.